exchanges/websocket/ob/ob.go

```diff
@@ -0,0 +1,235 @@
+package ob
+
+import (
+	"errors"
+	"fmt"
+	"sort"
+	"sync"
+
+	"github.com/thrasher-/gocryptotrader/currency"
+	"github.com/thrasher-/gocryptotrader/exchanges/orderbook"
+)
+
+const wsOrderbookBufferLimit = 5
```

Comment:
It would be nice to make this exportable so it can be adjusted on start-up for differing machine specs and to fine tune performance